test/local/log.js

```diff
@@ -462,29 +462,137 @@ describe('log', () => {
   it('.summary should log an info message and call request.emitRouteFlowEvent', () => {
     const emitRouteFlowEvent = sinon.spy()
     log.summary({
-      app: {},
+      app: {
+        accountRecreated: false,
+        acceptLanguage: 'en',
+        remoteAddressChain: ['95.85.19.180', '78.144.14.50']
+      },
+      auth: {
+        credentials: {
+          email: 'quix',
+          uid: 'quid',
+        }
+      },
       emitRouteFlowEvent: emitRouteFlowEvent,
-      headers: {},
+      headers: {
+        'user-agent': ''
```

Comment:
Should we put a value in for header so we can test that it is not empty string in assert?

Reply:
You are probably right.